<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VIPAdministration</class>
 <widget class="QDialog" name="VIPAdministration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>会员卡信息管理</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>400</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>400</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>640</width>
     <height>400</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <property name="minimumSize">
     <size>
      <width>640</width>
      <height>400</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>640</width>
      <height>400</height>
     </size>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <attribute name="title">
     <string>会员添加</string>
    </attribute>
    <widget class="QLabel" name="card">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>卡号</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="card_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="score">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>210</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>积分</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="name">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>姓名</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="phoneNumber">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>电话号码</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="idCard">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>身份证号</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="score_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>210</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="phoneNumber_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>170</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>130</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="idCard_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>90</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="add">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>280</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rewrite">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>280</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>重置</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <property name="minimumSize">
     <size>
      <width>640</width>
      <height>400</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>640</width>
      <height>400</height>
     </size>
    </property>
    <attribute name="title">
     <string>修改与删除</string>
    </attribute>
    <widget class="QLabel" name="card_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>卡号</string>
     </property>
    </widget>
    <widget class="QLabel" name="idCard_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>身份证号</string>
     </property>
    </widget>
    <widget class="QLabel" name="score_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>210</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>积分</string>
     </property>
    </widget>
    <widget class="QLabel" name="phoneNumber_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>电话号码</string>
     </property>
    </widget>
    <widget class="QLabel" name="name_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>姓名</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="score_4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>210</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>130</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>280</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>280</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="card_4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="idCard_4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>90</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="phoneNumber_4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>170</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
